lbrynet/cli.py

```diff
@@ -145,7 +145,12 @@ def main(argv=None):
         if method not in Daemon.deprecated_methods:
             print('{} is not a valid command.'.format(method))
             return 1
+
         new_method = Daemon.deprecated_methods[method].new_command
+        if new_method is None:
+            print("{} is permanently deprecated and will be removed in a future release.".format(method))
```

Comment:
This returns 0 but the message says "will be removed in a future release". If the command isn't run then isn't it basically removed now?

Reply:
Basically, maybe..!! But if you ponder into the deep ponds of philosophy and the definition of what existence means can one really say that something does or does not exist? 😂

On a less serious note: The command is still callable(though deprecated and doesn't return anything) so it is not completely removed(we can remove it completely if needed). When calling the command is an error then it will be said to be completely removed.

(Please do not solve your philosophical questions related to existence using this argument, some things are different for programs as compared to philosophy)

Reply:
The message should then say: `"{} is permanently deprecated and does not have a replacement command."`